ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -227,7 +239,6 @@ public synchronized void setText(String text, boolean animate) {
         }
 
         columnManager.setText(targetText, animate);
-        columnManager.setAnimationProgress(animate ? 0f : 1f);
```

Comment:
Yeah this is intentional as the animator should call `setAnimationProgress(0f)`.